src/output.c

```diff
@@ -388,6 +415,14 @@ output_init(struct server *server)
 	wl_list_init(&server->outputs);
 
 	output_manager_init(server);
+
+	if (getenv("LABWC_FALLBACK_OUTPUT")) {
+		output_add_virtual(server, "HEADLESS");
```

Comment:
The name prevents wayvnc from automatically resizing the output:
https://github.com/any1/wayvnc/blob/master/src/output.c#L222

Reply:
Happy to change the name - any alternative suggestions?

Reply:
On Wayfire, the fallback output is called NOOP-1, (for "no output") - would that name be ok?

Reply:
Anything starting with `HEADLESS-` or `NOOP-` would be fine for wayvnc.